### ISSUE TITLE
sonarjs/no-duplicated-branches doesn't report when switch case branches are identical

### ISSUE THREAD

**Issue by aprilandjan** [SonarSource/eslint-plugin-sonarjs#174]
**I want to report a bug.**

The rule [sonarjs/no-duplicated-branches](https://github.com/SonarSource/eslint-plugin-sonarjs/blob/master/docs/rules/no-duplicated-branches.md) doesn't report when:

1. using `switch` `case` statement;
2. the branch implementation has only one line of code;
3. the branch is identical with another branch.

**Reproducer**

```javascript
function first() {
  //
}

function second() {
  //
}

const v = Math.random();

switch(v) {
  case 0:
    first();
    break;
  case 1: // this branch is identical with the previous, but the rule gives no error unexpectedly
    first();
    break;
  case 2:
    first();
    second();
    break;
  case 3: // this branch is identical with the previous, the rule correctly gives an error as expected
    first();
    second();
  default:
}

if (v === 0) {
  first();
} else if (v === 1) { // in the meantime, this branch, with only one line of implementation, identical with the previous, the rule gives an error as expected
  first();
}

```

**Comment by Wohops** [comment]
This issue has been migrated to Jira. [ESLINTJS-7](https://sonarsource.atlassian.net/browse/ESLINTJS-7)
